tests/sqlparser_clickhouse.rs

```diff
@@ -394,3 +394,8 @@ fn clickhouse_and_generic() -> TestedDialects {
         options: None,
     }
 }
+
+#[test]
+fn parse_create_parametrised_views() {
+    clickhouse().verified_stmt("CREATE VIEW view AS SELECT * FROM A WHERE Column1 = {column1:datatype1} AND Column2 = {column2:datatype2}");
```

Comment:
Can you also please test:
1. In generic dialect
2. A single column definition
3. Error case (like `where Column1 = {column1}`)
4. Error case (like `where`) (no column definition)

Reply:
Sure, I'll add these tommorrow